build.gradle.kts

```diff
@@ -31,25 +29,15 @@ buildscript {
 }
 
 plugins {
-  id("com.autonomousapps.dependency-analysis") version libs.versions.dependencyAnalysisPlugin.get()
-  id("org.jetbrains.kotlinx.binary-compatibility-validator") version libs.versions.kotlinBinaryCompatibilityPlugin.get()
-}
-
-apply(plugin = "com.vanniktech.maven.publish.base")
-
-allprojects {
-  group = when {
```

Comment:
Is this group stanza unnecessary?

Reply:
I moved it into the settings script using a new API available in gradle 8.8!